src/store/slices/productFilters/index.ts

```diff
@@ -0,0 +1,72 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import { staticProductList } from "../../../mock-data";
+import { IProductFilters, filterOptionsDefault, productSortTypes } from "./model";
+
+const initialState: IProductFilters = {
+  filteredProductList: staticProductList,
+  referenceProductList: staticProductList,
+  filterOptions: filterOptionsDefault,
+};
+
+export const productFiltersSlice = createSlice({
+  name: "productFilters",
+  initialState,
+  reducers: {
+    setPriceFilter(state, action: PayloadAction<{ min: number; max: number }>) {
+      state.filterOptions.price = action.payload;
+    },
+    setSizeFilter(state, action: PayloadAction<string>) {
```

Comment:
Based on 'checked' field from input we can call 'push' or 'filter' without 'includes' condition.